index.md

```diff
@@ -33,35 +33,38 @@ Here's a simple program:
 
 ```val
 subscript longer_of(_ a: inout String, _ b: inout String): String {
-  yield if b.count() > a.count() { &b } else { &a }
+  if b.count() > a.count() { yield &b } else { yield &a }
+}
+
+func emphasize(_ z: inout String, strength: Int = 1) {
+  z.append(repeatElement("!", count: strength)))
```

Comment:
I would prefer if we stick to snake_case in the standard library.

Reply:
Sorry, just forgot.  Use of suggested change feature is generally appreciated when possible

Reply:
```suggestion
  z.append(repeat_element("!", count: strength)))
```